druid/examples/calc.rs

```diff
@@ -145,15 +150,19 @@ fn op_button(op: char) -> impl Widget<CalcState> {
 fn digit_button(digit: u8) -> impl Widget<CalcState> {
     let painter = Painter::new(|ctx, _, env| {
         let bounds = ctx.size().to_rect();
+        let color = env.get(theme::BACKGROUND_LIGHT);
+        let color_hot: Color = Color::from_rgba32_u32(color.clone().as_rgba_u32() + 0x303030);
```

Comment:
I like the idea of deriving a color from another color, but it feels a little verbose for an example. Is this something we should consider making easier in Piet, or at least refactoring as a function in here?

Reply:
Totally for adding a method in piet.

Reply:
Until then let's keep this as it is?